src/base/QXmppMessage.cpp

```diff
@@ -86,7 +88,9 @@ static const QStringList ENCRYPTION_NAMES = {
     QStringLiteral("OTR"),
     QStringLiteral("Legacy OpenPGP"),
     QStringLiteral("OpenPGP for XMPP (OX)"),
-    QStringLiteral("OMEMO")
+    QStringLiteral("OMEMO"),
+    QStringLiteral("OMEMO since version 0.4"),
+    QStringLiteral("OMEMO since version 0.8")
```

Comment:
```suggestion
    QStringLiteral("OMEMO:1"),
    QStringLiteral("OMEMO:2")
```
This is the name as it should be displayed in the UI, so a short name would be good. "OMEMO v0.8+" would also be an option, but I think the XEP versioning scheme isn't so good here, because the 0.x version number also changes when the encryption stays compatible.

Reply:
Okay, no actually this must be the name from the XEP, so "OMEMO 1" and "OMEMO 2".